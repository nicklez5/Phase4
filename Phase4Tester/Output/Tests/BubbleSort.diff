--- SelfTestCases/BubbleSort.out	2011-04-28 14:55:21.000000000 -0700
+++ ./Output/Tests/BubbleSort.out	2020-03-14 13:04:24.941787396 -0700
@@ -1,23 +1,12 @@
-20
-7
-12
-18
-2
-11
-6
-9
-19
-5
-99999
-2
-5
-6
-7
-9
-11
-12
-18
-19
-20
-0
+Error in /home/jackson/Desktop/phase4/Phase4Tester/./Output/Tests/BubbleSort.compiler.out line 261 column 11: "0": operand is of incorrect type
+Error in /home/jackson/Desktop/phase4/Phase4Tester/./Output/Tests/BubbleSort.compiler.out line 275 column 11: "1": operand is of incorrect type
+Error in /home/jackson/Desktop/phase4/Phase4Tester/./Output/Tests/BubbleSort.compiler.out line 289 column 11: "2": operand is of incorrect type
+Error in /home/jackson/Desktop/phase4/Phase4Tester/./Output/Tests/BubbleSort.compiler.out line 303 column 11: "3": operand is of incorrect type
+Error in /home/jackson/Desktop/phase4/Phase4Tester/./Output/Tests/BubbleSort.compiler.out line 317 column 11: "4": operand is of incorrect type
+Error in /home/jackson/Desktop/phase4/Phase4Tester/./Output/Tests/BubbleSort.compiler.out line 331 column 11: "5": operand is of incorrect type
+Error in /home/jackson/Desktop/phase4/Phase4Tester/./Output/Tests/BubbleSort.compiler.out line 345 column 11: "6": operand is of incorrect type
+Error in /home/jackson/Desktop/phase4/Phase4Tester/./Output/Tests/BubbleSort.compiler.out line 359 column 11: "7": operand is of incorrect type
+Error in /home/jackson/Desktop/phase4/Phase4Tester/./Output/Tests/BubbleSort.compiler.out line 373 column 11: "8": operand is of incorrect type
+Error in /home/jackson/Desktop/phase4/Phase4Tester/./Output/Tests/BubbleSort.compiler.out line 387 column 11: "9": operand is of incorrect type
 
+Processing terminated due to errors.
