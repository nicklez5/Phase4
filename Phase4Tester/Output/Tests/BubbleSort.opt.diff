--- SelfTestCases/BubbleSort.opt.out	2011-04-28 14:55:21.000000000 -0700
+++ ./Output/Tests/BubbleSort.opt.out	2020-03-14 13:04:23.501787434 -0700
@@ -1,23 +1,12 @@
-20
-7
-12
-18
-2
-11
-6
-9
-19
-5
-99999
-2
-5
-6
-7
-9
-11
-12
-18
-19
-20
-0
+Error in /home/jackson/Desktop/phase4/Phase4Tester/./Output/Tests/BubbleSort.opt.compiler.out line 244 column 11: "0": operand is of incorrect type
+Error in /home/jackson/Desktop/phase4/Phase4Tester/./Output/Tests/BubbleSort.opt.compiler.out line 258 column 11: "1": operand is of incorrect type
+Error in /home/jackson/Desktop/phase4/Phase4Tester/./Output/Tests/BubbleSort.opt.compiler.out line 272 column 11: "2": operand is of incorrect type
+Error in /home/jackson/Desktop/phase4/Phase4Tester/./Output/Tests/BubbleSort.opt.compiler.out line 286 column 11: "3": operand is of incorrect type
+Error in /home/jackson/Desktop/phase4/Phase4Tester/./Output/Tests/BubbleSort.opt.compiler.out line 300 column 11: "4": operand is of incorrect type
+Error in /home/jackson/Desktop/phase4/Phase4Tester/./Output/Tests/BubbleSort.opt.compiler.out line 314 column 11: "5": operand is of incorrect type
+Error in /home/jackson/Desktop/phase4/Phase4Tester/./Output/Tests/BubbleSort.opt.compiler.out line 328 column 11: "6": operand is of incorrect type
+Error in /home/jackson/Desktop/phase4/Phase4Tester/./Output/Tests/BubbleSort.opt.compiler.out line 342 column 11: "7": operand is of incorrect type
+Error in /home/jackson/Desktop/phase4/Phase4Tester/./Output/Tests/BubbleSort.opt.compiler.out line 356 column 11: "8": operand is of incorrect type
+Error in /home/jackson/Desktop/phase4/Phase4Tester/./Output/Tests/BubbleSort.opt.compiler.out line 370 column 11: "9": operand is of incorrect type
 
+Processing terminated due to errors.
