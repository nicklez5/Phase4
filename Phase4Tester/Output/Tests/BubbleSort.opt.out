Error in /home/jackson/Desktop/phase4/Phase4Tester/./Output/Tests/BubbleSort.opt.compiler.out line 244 column 11: "0": operand is of incorrect type
Error in /home/jackson/Desktop/phase4/Phase4Tester/./Output/Tests/BubbleSort.opt.compiler.out line 258 column 11: "1": operand is of incorrect type
Error in /home/jackson/Desktop/phase4/Phase4Tester/./Output/Tests/BubbleSort.opt.compiler.out line 272 column 11: "2": operand is of incorrect type
Error in /home/jackson/Desktop/phase4/Phase4Tester/./Output/Tests/BubbleSort.opt.compiler.out line 286 column 11: "3": operand is of incorrect type
Error in /home/jackson/Desktop/phase4/Phase4Tester/./Output/Tests/BubbleSort.opt.compiler.out line 300 column 11: "4": operand is of incorrect type
Error in /home/jackson/Desktop/phase4/Phase4Tester/./Output/Tests/BubbleSort.opt.compiler.out line 314 column 11: "5": operand is of incorrect type
Error in /home/jackson/Desktop/phase4/Phase4Tester/./Output/Tests/BubbleSort.opt.compiler.out line 328 column 11: "6": operand is of incorrect type
Error in /home/jackson/Desktop/phase4/Phase4Tester/./Output/Tests/BubbleSort.opt.compiler.out line 342 column 11: "7": operand is of incorrect type
Error in /home/jackson/Desktop/phase4/Phase4Tester/./Output/Tests/BubbleSort.opt.compiler.out line 356 column 11: "8": operand is of incorrect type
Error in /home/jackson/Desktop/phase4/Phase4Tester/./Output/Tests/BubbleSort.opt.compiler.out line 370 column 11: "9": operand is of incorrect type

Processing terminated due to errors.
